objects/_modal.scss

```diff
@@ -58,18 +57,11 @@ $modal-min-width: 160px;
 
 .modal__box {
     @include border-radius($modal-border-radius);
-    .is-cssexp & {
-        @include border-radius($exp-modal-border-radius);
-
-        // ensure border-radius clips inner content
-        overflow:hidden;
-    }
-
+    // ensure border-radius clips inner content
+    overflow:hidden;
+    color: $dark;
```

Comment:
TODO: check where this is used.

Reply:
Just seeing this line isn't necessary. `color: $modal-text` is a few lines below and that variable is set to `$dark` above.